test/require-config.js

```diff
@@ -6,30 +6,31 @@ require.config({
     baseUrl: '/base',
     waitSeconds: 60,
     paths: {
-        backbone: 'node_modules/backbone/backbone',
+        'backbone': 'node_modules/backbone/backbone',
```

Comment:
So has this change to ```consistent-as-needed```? And if so doesn't that mean that quotes can be removed when not needed (i.e. everything apart from 'backbone.paginator'?